HW4/Task42/RationalNumber.cs

```diff
@@ -0,0 +1,104 @@
+﻿namespace Task42
+{
+    internal sealed class RationalNumber : IComparable<RationalNumber>
+    {
+        public int Numerator { get; }
+        public int Denominator { get; }
+
+        // Constructor
+        public RationalNumber(int numerator, int denominator)
+        {
```

Comment:
denominator < 0 is normal behavior just change signs of denominator and numerator in this case (make denominator positive)